### PR TITLE
feat: Add new selenium action

### PR DESCRIPTION
**Pull request by nbrouand**

### CODE REVIEW

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyActionExtensions.kt [hunk 1]
```diff
@@ -1119,46 +1119,112 @@ fun ChutneyStepBuilder.SqlAction(
...
+
 /**
- * Start a local selenium driver instance
+ * Start a local or remote selenium driver instance with Chrome Driver
  * -------
  * Outputs:
  * - webDriver : instance of webdriver (org.openqa.selenium.WebDriver)
  * -------
- * Finally action registered : SeleniumQuitAction
+ * Finally action registered : SeleniumQuitTask
```
Comment (KarimGl) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyActionExtensions.kt:1141]: [SeleniumQuitAction](https://github.com/chutney-testing/chutney/blob/master/action-impl/src/main/java/com/chutneytesting/action/selenium/SeleniumQuitAction.java)

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyActionExtensions.kt [hunk 2]
```diff
@@ -1119,46 +1119,112 @@ fun ChutneyStepBuilder.SqlAction(
...
-
-enum class SELENIUM_BY {
-    id, NAME, className, cssSelector, xpath, zk;
+/**
+ * Start a local or remote selenium driver instance with Firefox Driver
+ * -------
+ * Outputs:
+ * - webDriver : instance of webdriver (org.openqa.selenium.WebDriver)
+ * -------
+ * Finally action registered : SeleniumQuitTask
```
Comment (KarimGl) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyActionExtensions.kt:1174]: SeleniumQuitAction

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyActionExtensions.kt [hunk 3]
```diff
@@ -1119,46 +1119,112 @@ fun ChutneyStepBuilder.SqlAction(
...
-            return v.name
-        }
-    }
+/**
+ * Start a remote selenium driver instance with json selenium configuration
+ * -------
+ * Outputs:
+ * - webDriver : instance of webdriver (org.openqa.selenium.WebDriver)
+ * -------
+ * Finally action registered : SeleniumQuitTask
```
Comment (KarimGl) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyActionExtensions.kt:1209]: SeleniumQuitAction

**Review by boddissattva — DISMISSED** [review]

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt [hunk 4]
```diff
@@ -1116,27 +1116,93 @@ fun ChutneyStepBuilder.SqlTask(
...
+ * Start a local or remote selenium driver instance with Chrome Driver
  * -------
  * Outputs:
  * - webDriver : instance of webdriver (org.openqa.selenium.WebDriver)
  * -------
  * Finally action registered : SeleniumQuitTask
  */
-fun ChutneyStepBuilder.SeleniumDriverInitTask(
-    browser: String? = null,
+fun ChutneyStepBuilder.SeleniumChromeDriverInitTask(
```
Comment (KarimGl) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt:1126]: Action not Task

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt [hunk 5]
```diff
@@ -1116,27 +1116,93 @@ fun ChutneyStepBuilder.SqlTask(
...
+    if (strategy != null) this.strategy = strategy
+}
+
+/**
+ * Start a local or remote selenium driver instance with Firefox Driver
+ * -------
+ * Outputs:
+ * - webDriver : instance of webdriver (org.openqa.selenium.WebDriver)
+ * -------
+ * Finally action registered : SeleniumQuitTask
```
Comment (KarimGl) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt:1157]: Action

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt [hunk 6]
```diff
@@ -1116,27 +1116,93 @@ fun ChutneyStepBuilder.SqlTask(
...
+    if (strategy != null) this.strategy = strategy
+}
+
+/**
+ * Start a remote selenium driver instance with json selenium configuration
+ * -------
+ * Outputs:
+ * - webDriver : instance of webdriver (org.openqa.selenium.WebDriver)
+ * -------
+ * Finally action registered : SeleniumQuitTask
```
Comment (KarimGl) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt:1192]: Action

**Review by KarimGl — APPROVED** [review]

**Pull request merged**